### ISSUE TITLE
[docs] The getting started guide is hard to follow

### ISSUE THREAD

**Issue by wagnermaciel** [material-components/material-components-web#6602]
### Steps to reproduce
1. Go to `material-components-web/docs/getting-started.md`.
2. Try to follow the guide.

### Actual behavior

You have to debug as you go along.

### Expected behavior

You don't have to debug as you go through the guide.

### Your Environment:

| Software         | Version(s) |
| ---------------- | ---------- |
| MDC Web          | 8.0.0 |
| Browser          | Google Chrome Version 86.0.4240.183 |
| Operating System | macOS Catalina

**Comment by asyncLiz** [comment]
Can you provide specifics as to what parts of the getting started guide are difficult to follow?

**Comment by dieseld23** [comment]
I tried to walk through the whole thing and it doesn't work. I think mainly due to version incompatibilities. npm install webpack installs webpack 5, which now uses 'webpack serve' instead of 'webpack-dev-server' (using webpack-dev-server causes some errors from the get go). Which version do you need to run this properly?

**Comment by joyzhong** [comment]
This was fixed in https://github.com/material-components/material-components-web/pull/6679, closing this bug.

Wagner, please feel free to reopen with more specific details of what steps were hard to follow.
